WordPress/gutenberg
# Component System: Upgrade FontSizePicker

This update adapts `FontSizePicker` from `@wordpress/components` with the new Component System setup.
It does this by adding the (current) external `@wp-g2` component system packages as dependencies and using them directly.

This strategy was something @gziolo had suggested here:
https://github.com/WordPress/gutenberg/issues/27484#issuecomment-738857333


## How has this been tested?

* `npm install`
* Run `npm run dev`
* Fire up local Gutenberg
* Edit or add a Paragraph block
* Adjust the font-size
* It should work like normal (and the controls may look slightly slightly different)

## Screenshots

![wp-g2-font-size-picker](https://user-images.githubusercontent.com/2322354/101541946-0ea8dc80-3970-11eb-8259-59f55e22d752.gif)


## Types of changes

* Adding `@wp-g2` dependencies
* Add `COMPONENT_SYSTEM_PHASE` env flag
* Created a context/adapter
* Connected `FontSizePicker` to the component system context

## Checklist:
